releases/nightly-build/compose-files/docker-compose-nexus-mongo-arm64.yml

```diff
@@ -440,6 +455,7 @@ services:
       - consul
       - logging
       - data
+      - vault-worker
```

Comment:
Technically not needed since the `rules-engine` profile doesn't require any secrets, but good to leave as an example for other instances that may use a profile that is dependent of secrets.